lib/screens/login/login.dart

```diff
@@ -22,13 +27,21 @@ class LoginScreen extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    SharedPreferences sharedPreferences = sl.get<SharedPreferences>();
     return Scaffold(
       resizeToAvoidBottomInset: false,
       body: BlocConsumer<LoginCubit, LoginState>(
         listener: (context, state) async {
           if (state is LoginSuccess) {
+            await sharedPreferences.setString(S.tokenKeySharedPreferences, state.token);
             await Future.delayed(Duration(seconds: 1));
```

Comment:
1. no need for this.
2. add a snackbar to tell the user login was successful.

Reply:
done